manifests/database/postgresql.pp

```diff
@@ -9,12 +9,14 @@
   $manage_server        = $puppetdb::params::manage_dbserver,
   $manage_package_repo  = $puppetdb::params::manage_pg_repo,
   $postgres_version     = $puppetdb::params::postgres_version,
+  $postgres_datadir     = $puppetdb::params::postgres_datadir,
```

Comment:
The thing about these kind of parameters that get proxied over to the postgresql module, is that you never win unless you do them all. They'll continue to be added and added by contributors until we have about 80% of them, never quite all of them.

I'm wary of adding any more of these to this module as really, if the functionality is required, one should be breaking out from this - and using the postgresql module independently instead. The other alternative is to provide a way to proxy everything (like passing a hash), without hard-coding defaults (like below) as well, so it acts as a really dumb proxy rather then having each parameter manually copied over.

But really, what is the value of having this as a convenience (when considering option 2) when really one can break out from using this class, and use postgresql directly instead.